cli/src/cli_util.rs

```diff
@@ -260,10 +260,23 @@ impl TracingSubscription {
     pub fn enable_debug_logging(&self) -> Result<(), CommandError> {
         self.reload_log_filter
             .modify(|filter| {
+                // the default is INFO
+                // jj-lib and jj-cli are whitelisted for DEBUG logging
+                // this ensures that other crates logging doesn't come int by default
                 *filter = tracing_subscriber::EnvFilter::builder()
-                    .with_default_directive(tracing::metadata::LevelFilter::DEBUG.into())
+                    .with_default_directive(tracing::metadata::LevelFilter::INFO.into())
```

Comment:
Is this the `tracing_subscriber` default? If so, we could drop this line, right?

Reply:
Ah, apparently there is no default.